init.go

```diff
@@ -34,7 +35,9 @@ func Run(m *testing.M, opts ...agent.Option) int {
 	newAgent, err := agent.NewAgent(opts...)
 	if err != nil {
 		res := m.Run()
-		fmt.Printf("\n%v\n", err)
+		if env.ScopeDebug.Value {
```

Comment:
Is this line preventing to show the message if `SCOPE_DSN` is not set? (previous message)

Reply:
This line is preventing to show any initialization message if the debug flag is not set.